bagitobjecttransfer/recordtransfer/templates/includes/profile_form.html

```diff
@@ -0,0 +1,60 @@
+{% load i18n %}
+{% include "includes/messages.html" %}
+<script>
+    // Pass constants to JavaScript
+    const ID_GETS_NOTIFICATION_EMAILS = "{{ ID_GETS_NOTIFICATION_EMAILS }}";
+    const ID_CURRENT_PASSWORD = "{{ ID_CURRENT_PASSWORD }}";
+    const ID_NEW_PASSWORD = "{{ ID_NEW_PASSWORD }}";
+    const ID_CONFIRM_NEW_PASSWORD = "{{ ID_CONFIRM_NEW_PASSWORD }}";
+</script>
```

Comment:
This will place your JS in the head of the document which would be preferred to having a random <script> in the middle of the document.

```suggestion
{% block javascript %}
    {{ block.super }}
    <script type="text/javascript">
        // Pass constants to JavaScript
        const ID_GETS_NOTIFICATION_EMAILS = "{{ ID_GETS_NOTIFICATION_EMAILS }}";
        const ID_CURRENT_PASSWORD = "{{ ID_CURRENT_PASSWORD }}";
        const ID_NEW_PASSWORD = "{{ ID_NEW_PASSWORD }}";
        const ID_CONFIRM_NEW_PASSWORD = "{{ ID_CONFIRM_NEW_PASSWORD }}";
    </script>
{% endblock %}
```

Reply:
Could you check if this works for you?

I'm getting back this rendering error:
```
Error during template rendering
In template /app/bagitobjecttransfer/recordtransfer/templates/includes/profile_form.html, error at line 4

'BlockNode' object has no attribute 'context'. Did you use {{ block.super }} in a base template?
1	{% load i18n %}
2	{% include "includes/messages.html" %}
3	{% block javascript %}
4	    {{ block.super }}

```

I think it might be because the `javascript` block is defined in `base.html`, which `profile.html` is extending. `profile_form.html` is included in `profile.html`, but maybe `profile_form.html` doesn't have access to `base.html`'s context unless it is directly extending it?

Reply:
If it's an issue with extending a block that doesn't exist in the extended form, maybe it would work to add the block in profile.html without changes.

```
# (in profile.html)
{% block javascript %}
    {{ block.super }}
{% endblock %}
```

Reply:
Resolved by:
https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/226/commits/bf1d9d5a012c5a2c091b4604b858c9350c06a4b3